src/scripts/browserDeviceProfile.js

```diff
@@ -294,12 +294,55 @@ import browser from './browser';
                     (browser.tizen && isTizenFhd ? 20000000 : null)));
     }
 
+    function getSpeakerCount() {
+        const AudioContext = window.AudioContext || window.webkitAudioContext || false; /* eslint-disable-line compat/compat */
+
+        if (AudioContext) {
+            const audioCtx = new AudioContext();
+
+            return audioCtx.destination.maxChannelCount;
```

Comment:
Testing this on the webOS 1.2 emulator, I got `maxChannelCount: 0` and `channelCount: 2`.
I'm not sure if adding ` || audioCtx.destination.channelCount` is right.
It is possible that this is an emulator bug, so it is probably better to just leave it for now.

Reply:
I believe this should not ever happen, according to [docs](https://webaudio.github.io/web-audio-api/#dom-audionode-channelcount):  
> ...some nodes have additional channelCount constraints on the possible values for the channel count:
> ... AudioContext
> The channel count MUST be between 1 and maxChannelCount. An IndexSizeError exception MUST be thrown for any attempt to set the count outside this range.